modules/monitoring-limiting-scrape-samples-in-user-defined-projects.adoc

```diff
@@ -2,19 +2,21 @@
 //
 // * monitoring/configuring-the-monitoring-stack.adoc
 
+:_content-type: CONCEPT
 [id="controlling-the-impact-of-unbound-attributes-in-user-defined-projects_{context}"]
 = Controlling the impact of unbound metrics attributes in user-defined projects
 
 Developers can create labels to define attributes for metrics in the form of key-value pairs. The number of potential key-value pairs corresponds to the number of possible values for an attribute. An attribute that has an unlimited number of potential values is called an unbound attribute. For example, a `customer_id` attribute is unbound because it has an infinite number of possible values.
 
-Every assigned key-value pair has a unique time series. The use of many unbound attributes in labels can result in an exponential increase in the number of time series created. This can impact Prometheus performance and can consume a lot of disk space.
+Every assigned key-value pair has a unique time series. The use of many unbound attributes in labels can result in an exponential increase in the number of time series created. This can impact Prometheus performance and can consume a high amount of disk space.
 
 Cluster administrators can use the following measures to control the impact of unbound metrics attributes in user-defined projects:
 
-* *Limit the number of samples that can be accepted* per target scrape in user-defined projects
-* *Create alerts* that fire when a scrape sample threshold is reached or when the target cannot be scraped
+* Limit the number of samples that can be accepted per target scrape in user-defined projects
+* Limit the number of scraped labels, the length of label names, and the length of label values.
+* Create alerts that fire when a scrape sample threshold is reached or when the target cannot be scraped
 
 [NOTE]
 ====
-Limiting scrape samples can help prevent the issues caused by adding many unbound attributes to labels. Developers can also prevent the underlying cause by limiting the number of unbound attributes that they define for metrics. Using attributes that are bound to a limited set of possible values reduces the number of potential key-value pair combinations.
+Limiting scrape samples and label names can help prevent issues caused by adding many unbound attributes to labels. Developers can also prevent the underlying cause by limiting the number of unbound attributes that they define for metrics. Using attributes that are bound to a limited set of possible values reduces the number of potential key-value pair combinations.
```

Comment:
```suggestion
To prevent issues caused by adding many unbound attributes, limit the number of scrape samples, label names, and unbound attributes you define for metrics. Also reduce the number of potential key-value pair combinations by using attributes that are bound to a limited set of possible values.
```
How about rewriting this to be more user-action focused?

Reply:
> To prevent issues caused by adding many unbound attributes, limit the number of scrape samples, label names, and unbound attributes you define for metrics. Also reduce the number of potential key-value pair combinations by using attributes that are bound to a limited set of possible values.

Edited per your suggested wording.